astroplan/core.py

```diff
@@ -0,0 +1,234 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..extern import six
+
+import sys
+from math import sqrt, pi, exp, log, floor
+from abc import ABCMeta, abstractmethod
+
+import numpy as np
+
+# Originally authored by Jazmin Berlanga Medina (jazmin.berlanga@gmail.com),
+# and modified by Adrian Price-Whelan (email) and Erik Tollerud (email).
+
+__all__ = ["Observer", "Date", "Target", "FixedTarget", "NonFixedTarget",
+           "Constraints", "Observation"]
+
+#__doctest_requires__ = {'*': ['scipy.integrate']}
+
+
+class Observer(object):
+    """
+    Some comments.
+    """
+    #def __init__(self, name, longitude, latitude, elevation, pressure,
+    #             rel_humidity, temp, timezone, description=None):
+    """
+    Comments.
+    """
+
+    #def set_environment():
+    """
+    Comments.
+    """
+
+    raise NotImplementedError()
+
+
+class Date(Observer):
```

Comment:
I'm not sure what you mean here: `Date` isn't a new class to be created, it already exists as `astropy.time.Time` - the idea is that anywhere that dates fit, anything that can be converted to a `Time` object should be used.